src/components/TodoItem.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import styled from "styled-components";
+import checked from "../assets/checked.svg";
+import unchecked from "../assets/unchecked.svg";
+import deleteBtn from "../assets/deleteBtn.svg";
+
+const TodoItem = ({ todo, date, onTodoChange }) => {
+
+  // 투두 완료 토글
+  const toggleTodoComplete = () => {
+    onTodoChange(date, todo, false);
+  };
+
+  // 투두 삭제
+  const deleteTodo = () => {
+    onTodoChange(date, todo, true);
+  };

```

Comment:
이 두함수는 공통요소가 2개이고, boolean값만 다른데, 하나의 함수로 통일해서 리팩토링하면 좋을 것 같아요

```suggestion
const handleAction = (action) => {
  onTodoChange({ date, todo, action });
};
```